src/pages/dan/Main/Main.js

```diff
@@ -1,10 +1,236 @@
-import React from "react";
+import React from 'react';
+import { Link } from 'react-router-dom';
 import { withRouter } from 'react-router-dom';
```

Comment:
- 다음과 같이 한줄에서 import 할 수 있습니다.
```suggestion
import { Link, withRouter } from 'react-router-dom';
```